client/layout/index.jsx

```diff
@@ -116,6 +110,13 @@ class Layout extends Component {
 		if ( prevProps.colorSchemePreference === this.props.colorSchemePreference ) {
 			return;
 		}
+
+		// This code should be removed when the nav-unification project has been rolled out to 100% of the customers.
```

Comment:
Moving this code block **first** will guarantee that it runs. There are several return early statements which can prevent from code reaching there.